api/db/seeds/data/team-certification/cases/sco-not-managing-students-certification.js

```diff
@@ -0,0 +1,188 @@
+import { Candidate } from '../../../../../src/certification/enrolment/domain/models/Candidate.js';
+import { Subscription } from '../../../../../src/certification/enrolment/domain/models/Subscription.js';
+import { usecases as enrolmentUseCases } from '../../../../../src/certification/enrolment/domain/usecases/index.js';
+import { OrganizationForAdmin } from '../../../../../src/organizational-entities/domain/models/OrganizationForAdmin.js';
+import { usecases as organizationalEntitiesUsecases } from '../../../../../src/organizational-entities/domain/usecases/index.js';
+import * as organizationCreationValidator from '../../../../../src/organizational-entities/domain/validators/organization-creation-validator.js';
+import {
+  CertificationCenter,
+  types as certificationCenterTypes,
+} from '../../../../../src/shared/domain/models/CertificationCenter.js';
+import { Membership } from '../../../../../src/shared/domain/models/index.js';
+import { normalize } from '../../../../../src/shared/infrastructure/utils/string-utils.js';
+import { usecases as teamUsecases } from '../../../../../src/team/domain/usecases/index.js';
+import * as tooling from '../../common/tooling/index.js';
+import { acceptPixOrgaTermsOfService } from '../../common/tooling/legal-documents.js';
+import {
+  CERTIFICATION_SCO_NOT_MANAGING_STUDENTS_EXTERNAL_ID,
+  SCO_CERTIFICATION_NOT_MANAGING_STUDENTS_CERTIFICATION_CENTER_USER_ID,
+  SCO_CERTIFICATION_NOT_MANAGING_STUDENTS_ORGANIZATION_USER_ID,
+  SCO_NOT_MANAGING_CERTIFICATION_CENTER_ID,
+} from '../constants.js';
+
+/**
+ * --- CERTIFICATION CASE ---
+ *
+ * The goal here is to reproduce one certification case:
+ *   - The organization is SCO but not managing students
+
+ *   - A session with several candidates
+ *   - One certifiable candidate is able to start a certification course
```

Comment:
```suggestion
 * The goal here is to reproduce the most simple certification case:
 *   - I'm a SCO user with a certifiable account
 *   - I'm able to start a certification course
 *   - The organization is SCO, not managing students
 *   - The certification session has multiple candidates.
```

On pourrait peut-être expliciter ce que ça engage ce "not managing students" ?
Dans les faits, c'est pas forcément super clair !

Reply:
on passe à du PRO